CMakeLists.txt

```diff
@@ -4,22 +4,11 @@ set(CMAKE_SYSTEM_NAME Generic)
 set(CMAKE_SYSTEM_VERSION 1)
 set(CMAKE_SYSTEM_PROCESSOR cortex-m3)
 
-set(SERIAL_PORT /dev/ttyACM0)
-
-if (WIN32)
-    set(OBJCOPY "C:/SysGCC/arm-eabi/bin/arm-eabi-objcopy")
-    set(CMAKE_C_COMPILER C:/SysGCC/arm-eabi/bin/arm-eabi-gcc.exe)
-    set(CMAKE_CXX_COMPILER C:/SysGCC/arm-eabi/bin/arm-eabi-g++.exe)
-elseif(APPLE)
-    set(OBJCOPY "/opt/arm-none-eabi/bin/arm-none-eabi-objcopy")
-    set(CMAKE_C_COMPILER "/opt/arm-none-eabi/bin/arm-none-eabi-gcc")
-    set(CMAKE_CXX_COMPILER "/opt/arm-none-eabi/bin/arm-none-eabi-g++")
-else ()
-    set(OBJCOPY "$ENV{HOME}/opt/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-objcopy")
-    set(CMAKE_C_COMPILER "$ENV{HOME}/opt/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-gcc")
-    set(CMAKE_CXX_COMPILER "$ENV{HOME}/opt/gcc-arm-none-eabi-7-2018-q2-update/bin/arm-none-eabi-g++")
-    set(TERMINAL "${CMAKE_CURRENT_SOURCE_DIR}/tools/lpc21isp_197/lpc21isp_hr_linux")
-    set(BOSSAC "bossac")
+if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/toolchains/toolchain.cmake")
```

Comment:
This is because cmake doesnt generate a error if the file doesnt exist.

[https://cmake.org/cmake/help/v3.0/command/include.html](https://cmake.org/cmake/help/v3.0/command/include.html)